modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1741,11 +1840,20 @@ unsigned int
 EFAElement3D::numFaceNeighbors(unsigned int face_id) const
 {
   unsigned int num_neighbors = 0;
-  if (_face_neighbors[face_id][0])
+  if (_face_neighbors[face_id].size())
     num_neighbors = _face_neighbors[face_id].size();
   return num_neighbors;
 }
 
+unsigned int
+EFAElement3D::numEdgeNeighbors(unsigned int face_id, unsigned int edge_id) const
+{
+  unsigned int num_neighbors = 0;
+  if (_face_edge_neighbors[face_id][edge_id].size())
+    num_neighbors = _face_edge_neighbors[face_id][edge_id].size();
+  return num_neighbors;
```

Comment:
Same with this function:
`return _face_edge_neighbors[face_id][edge_id].size();`